products/citrix-apps-desktops.md

```diff
@@ -15,6 +15,10 @@ releaseDateColumn: true
 sortReleasesBy: releaseDate
 LTSLabel: "<abbr title='Long Term Service Release'>LTSR</abbr>"
 releases:
+-   releaseCycle: "2206 (CR)"
+    support: 2022-12-28
+    eol: 2023-12-28
+    releaseDate: 2022-06-28
```

Comment:
https://docs.citrix.com/en-us/citrix-virtual-apps-desktops/whats-new.html says release date was 30th June.
https://www.citrix.com/downloads/citrix-virtual-apps-and-desktops/ says 28th June.

The RSS feed also says 28th, so going ahead and merging 👍🏽